app/controllers/api/v2/base_controller.rb

```diff
@@ -64,6 +66,10 @@ def metadata_total
         @total ||= resource_scope.try(:count).to_i
       end
 
+      def can_create?
+        authorized_for(controller: controller_permission, action: 'create', user_id: User.current)
```

Comment:
If we have an authorizer already created for the request you can reuse it for this as well